haxepunk/Screen.hx

```diff
@@ -324,23 +255,6 @@ class Screen
 	function get_mouseY():Int return Std.int(_sprite.mouseY);
 
 	/**
-	 * Captures the current screen as an Image object.
-	 * Will only work in buffer rendermode (flash or html5).
-	 * @return	A new Image object.
-	 */
-	public function capture():Image
```

Comment:
This was something I considered as well. Especially since you've made render targets fairly easy to create.